src/js/pages/system/OverviewDetailTab.js

```diff
@@ -30,7 +31,7 @@ class OverviewDetailTab extends mixin(StoreMixin) {
       open: false
     };
 
-    this.store_listeners = [
+    this.store_listeners = Hooks.applyFilter('OverviewDetailTab:StoreListeners', [
```

Comment:
I like the namespacing, perhaps that's something we should use to all other components. If we decided to do so, please create a dech-debt jira for this.

However for the hook itself, so far we are using lowercase first letter. So I would go for `OverviewDetailTab:storeListeners`